pkg/controllers/managementuser/rbac/prtb_handler.go

```diff
@@ -297,3 +306,63 @@ func buildRule(resource string, verbs map[string]bool) rbacv1.PolicyRule {
 		APIGroups: []string{"*"},
 	}
 }
+
+func (p *prtbLifecycle) reconcilePRTBUserClusterLabels(binding *v3.ProjectRoleTemplateBinding) error {
+	/* Prior to 2.5, for every PRTB, following CRBs are created in the user clusters
+		1. PRTB.UID is the label key for a CRB, PRTB.UID=owner-user
+		2. PRTB.UID is the label value for RBs with authz.cluster.cattle.io/rtb-owner: PRTB.UID
+	Using this labels, list the CRBs and update them to add a label with ns+name of CRTB
+	*/
+
+	set := labels.Set(map[string]string{string(binding.UID): owner})
+	userCRBs, err := p.m.clusterRoleBindings.List(metav1.ListOptions{LabelSelector: set.String()})
```

Comment:
Look into doing inequality in the list, that will stop from getting all the ones you already updated back in the response https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#equality-based-requirement

Reply:
good point! will add this in the next commit